examples/with-aws-serverless/serverless.yml

```diff
@@ -0,0 +1,51 @@
+service: nextjs-aws-serverless
+
+package:
+  exclude:
+    - pages/**
+    - tests/**
+    - node_modules/.cache/**
+    - .cache
+    - package.json
+    - package-lock.json
+    - "**.test.js"
+    - "__snapshots__/"
+  include:
+    - .next/**
+
+plugins:
+  - serverless-offline
+  - serverless-apigw-binary
+
+custom:
+  serverless-offline:
+    port: 8000
+    dontPrintOutput: true
+  apigwBinary:
+    types:
+      - '*/*'
+
+provider:
+  name: aws
+  runtime: nodejs6.10
+  memorySize: 256
+  timeout: 15
+  environment:
+    NODE_ENV: production
+
+functions:
+  nextJsRootApp:
+    handler: lambda.handler
+    events:
```

Comment:
You can merge `nextJsRootApp` and `nextJsAnyApp` events - just change the events to this:

```diff
    events:
     - http: ANY {proxy+}
+    - http: ANY /
```